### ISSUE TITLE
No ALT tags on amp-YouTube cover images hurting Google SEO/Accessibility Score

### ISSUE THREAD

**Issue by Kist221** [ampproject/amphtml#13958]
So I have an AMP page with some Youtube embeds using the documented amp-youtube tag.

My AMP code:

<amp-youtube data-videoid="XXPNAEoX70w" layout="responsive" width="480" height="270"></amp-youtube>
Rendered child code:

<div class="ytp-cued-thumbnail-overlay-image" style="background-image: url(&quot;https://i.ytimg.com/vi_webp/XXPNAEoX70w/maxresdefault.webp&quot;);"></div>
A background image is automatically rendered through the AMP js without an ALT tag that hurts Google's accessibility score ironically.

Adding an alt tag to parent items doesn't work so I'm kind of stumped at how to resolve this in the interim of Google getting their act together.

Clever ideas?

**Comment by aghassemi** [comment]
Hi @Kist221. Thanks for raising the bug. Please see https://github.com/ampproject/amphtml/issues/5508 which is same as your issue.

We have made https://github.com/ampproject/amphtml/issues/5508 into a "good first issue" with plenty of instructions on how to contribute a fix to AMP, if this is something that you are interested in contributing, we would appreciate it!

**Comment by aghassemi** [comment]
Closing as dup of https://github.com/ampproject/amphtml/issues/5508 lets have follow up conversations there.
